package.json

```diff
@@ -44,6 +44,7 @@
     "generate": "nx generate",
     "get-secrets": "./scripts/get-secrets.sh",
     "create-secret": "./scripts/create-secret.sh",
+    "proxies": "./scripts/run-test-proxies.sh",
```

Comment:
This PR deletes the `run-test-proxies.sh` file. Please update this command.

Reply:
Updated the command :)